From d9ff22d53e2173db45ff0bf07d7e59f17974f4c9 Mon Sep 17 00:00:00 2001
From: Ryan Govostes <rgov@users.noreply.github.com>
Date: Thu, 3 Dec 2020 21:27:18 -0500
Subject: [PATCH] cmake: Fix variable substitution in pkg-config

Variable interpolation is done with configure_file(... @ONLY), so use @VAR@ and not ${VAR}.

Fixes #3969.
---
 gr-qtgui/gnuradio-qtgui.pc.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gr-qtgui/gnuradio-qtgui.pc.in b/gr-qtgui/gnuradio-qtgui.pc.in
index 2489caf2141..a1e93eb4712 100644
--- gr-qtgui/gnuradio-qtgui.pc.in
+++ gr-qtgui/gnuradio-qtgui.pc.in
@@ -7,5 +7,5 @@ Name: gnuradio-qtgui
 Description: GNU Radio blocks for QT GUI
 Requires: gnuradio-runtime
 Version: @LIBVER@
-Libs: -L${libdir} -lgnuradio-qtgui ${PC_LIBS_STR}
-Cflags: -I${includedir} -I${QWT_INCLUDE_DIRS} -I${QT_HEADERS_DIR}
+Libs: -L${libdir} -lgnuradio-qtgui @PC_LIBS_STR@
+Cflags: -I${includedir} -I@QWT_INCLUDE_DIRS@ -I@QT_HEADERS_DIR@
